### PR TITLE
tenant requested changes and make description render as html

### PR DESCRIPTION
**Pull request by sarveshpro**
Changes in this pull request:
- tenant requested changes
- make description render as HTML

**Warning: Before merging this hotfix please test with other tenants using the common homepage, this may be a breaking change.**

### CODE REVIEW

**Review by norbertschuler — APPROVED** [review]
LGFM

File: public/tenants/culchacandela/config.ts [hunk 1]
```diff
@@ -22,12 +22,12 @@ const config = {
   home: {
     title: 'Culcha Candela X Plant-for-the-Planet ',
     description:
-      'Wir pflanzen Bäume für ein besseres Weltklima, um uns so wichtige Zeit im Wettlauf gegen die Klimakrise zu verschaffen. Pflanz’ hier mit! Und beim Hören des neuen „Hamma (INVCTS & Dorfkind J-P Remix)” auf music-for-nature & sämtlichen Musik Streaming-Plattformen. Die Streaming Einnahmen spenden wir an Plant-for-the-Planet!',
+      '<p>Wir pflanzen B&auml;ume f&uuml;r ein besseres Weltklima, um uns so wichtige Zeit im Wettlauf gegen die Klimakrise zu verschaffen. Pflanz&rsquo; hier mit!&nbsp;<br />&hellip;und beim H&ouml;ren des neuen &bdquo;<a href="https://ffm.to/hamma-4-the-planet" target="_blank" rel="noopener noreferrer">Hamma (INVCTS &amp;Dorfkind J-P Remix)</a>&rdquo;&nbsp;auf&nbsp;s&auml;mtlichen Musik Streaming-Plattformen.<br />Wir spenden die kompletten Streaming Einnahmen des Songs an Plant-for-the-Planet! Powered by&nbsp;<a href="https://www.music-for-nature.net/" target="_blank" rel="noopener noreferrer">music-for-nature</a>!</p>',
```
Comment (norbertschuler) [inline public/tenants/culchacandela/config.ts:25]: Watch out for the merge conflict we will get with https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/817 where the description is extracted into a JSON file not containing any HTML. Maybe HTML is also not allowed in i18n files and therefore not working - we should check that.

**Comment by harshvitra** [comment]
Can it be done using i18n Trans Component ?

**Comment by norbertschuler** [comment]
> Can it be done using i18n Trans Component ? @norbertschuler @sarveshpro

Not sure. I asked you the same at https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/817#discussion_r578272825 ;-)

File: public/tenants/culchacandela/config.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ const config = {
   tenantURL: 'wald.culchacandela.de',
   languages: ['en', 'de'],
   fallbackCurrency: 'EUR',
-  tenantGoal: null,
+  tenantGoal: 77.777,
```
Comment (norbertschuler) [inline public/tenants/culchacandela/config.ts:6]: Requested to be changed to 77.700 now.

**Comment by norbertschuler** [comment]
Instead of merging this branch we could also deploy it directly to Netlify instead of the main branch for culchacandela. We just have to care about it later.

**Comment by norbertschuler** [comment]
This branch is now deployed as https://wald.culchacandela.de/home - do not change it (except for fixes).

**Comment by norbertschuler** [comment]
> This branch is now deployed as https://wald.culchacandela.de/home - do not change it (except for fixes). I added a protection to it, so nobody accidentally deletes it.

I decided to rather create a new branch `tenant/culchacandela` again and use it for deployment instead of this branch. Currently both branches are the same commit, but I think it's cleaner that way.

**Comment by harshvitra** [comment]
> I decided to rather create a new branch `tenant/culchacandela` again and use it for deployment instead of this branch. Currently both branches are the same commit, but I think it's cleaner that way. So whatever we do here, does not affect the deployment any more.

In that case can we close PR (and maybe also delete this branch)

**Pull request closed without merging**